examples/spec_with_photo.py

```diff
@@ -0,0 +1,498 @@
+"""
+NUTS analysis of the high-resolution emission spectrum with photometric information
+====================================================================================
+
+This script performs a Bayesian analysis of the high-resolution emission spectrum with photometric information.
+
+"""
+
+# from jax import config
+# config.update("jax_enable_x64", True)
+
+import numpy as np
+import jax.numpy as jnp
+import matplotlib.pyplot as plt
+from exojax.spec.specop import SopPhoto
+from exojax.test.emulate_spec import sample_emission_spectrum
+from exojax.utils.grids import wavenumber_grid
+from exojax.utils.instfunc import resolution_to_gaussian_std
+
+from jax import jit
+
```

Comment:
This is just a suggestion, but to ensure the existence of the output directories
```suggestion
+import os
+os.makedirs("./output", exist_ok=True)
+os.makedirs("./output_bn", exist_ok=True)
```